ember-toucan-core/src/components/form/input-field.hbs

```diff
@@ -0,0 +1,32 @@
+<div
+  class="flex flex-col {{if @isDisabled 'text-disabled cursor-not-allowed'}}"
+>
+  <Form::Field as |field|>
+    <field.Label for={{field.id}}>{{@label}}</field.Label>
+
+    {{#if @hint}}
+      <field.Hint id={{field.hintId}}>{{@hint}}</field.Hint>
+    {{/if}}
+
+    <field.Control class="flex">
+      <input
+        class="m-1 h-6 w-0 flex-grow bg-transparent p-1
+          {{if
+            @isDisabled
+            'text-disabled cursor-not-allowed'
+            'text-titles-and-attributes'
+          }}
+          "
+        id={{field.id}}
+        aria-describedby="{{if @hint field.hintId}} {{if @error field.errorId}}"
+        aria-invalid={{if @error "true"}}
+        disabled={{@isDisabled}}
+        ...attributes
```

Comment:
Kind of wondering if it's worth it for us to set `type="text"` by default?  Then folks can override since we `...attributes`.  What do you think?